src/main/java/homework_5/custom_regex_matcher/CustomRegexMatcher.java

```diff
@@ -0,0 +1,27 @@
+package homework_5.custom_regex_matcher;
+
+import java.util.regex.Pattern;
+
+import static java.lang.System.lineSeparator;
+
+public class CustomRegexMatcher {
+    private static final String CUSTOM_REGEX = "\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}:\\d+?[zZ]\\s*–\\s*\\[.*]\\s*-\\s*.*";
+    public static final String ERROR_MESSAGE = "No arguments have been found";
```

Comment:
ofc it's a good idea to see such notification, but following the task your app should just print only true/false.